apps/pano/app/features/post/PopularPostsTimeframeFilters.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  OldButton,
+  DropdownMenuContent,
+} from "@kampus/ui";
+import { Form, useLocation, useNavigation } from "@remix-run/react";
+import { useEffect, useState } from "react";
+
+export const TIME_FRAMES = [
+  { text: "Now", value: "now" },
+  { text: "Today", value: "today" },
+  { text: "This Week", value: "week" },
+  { text: "This Month", value: "month" },
+  { text: "This Year", value: "year" },
+  { text: "All Time", value: "all" },
+] as const;
+
+export const PopularPostsTimeframeFilters = () => {
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+
+  const navigation = useNavigation();
+  const location = useLocation();
+  const timeframe =
+    new URLSearchParams(location.search).get("timeframe") || "all";
```

Comment:
Can you check the Hacker News' implementation? They use `day` search parameter and the value is the starting date. I think their approach makes more sense

Reply:
Agreed, i think we need to first talk about the tech design in #304, let's move this discussion over there.

Reply:
🙏